content/applications/productivity/spreadsheet/global_filters.rst

```diff
@@ -0,0 +1,203 @@
+==============
+Global filters
+==============
+
+.. role:: raw-html(raw)
+   :format: html
+
+.. |global-filter| replace:: :raw-html:`<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewbox="0 0 20 20"><path fill="currentColor" d="M1 3h12L7 9M5.5 6h3v11l-3-3M14 4h4v2h-4m-3 3h7v2h-7m0 3h7v2h-7"></path></svg>` :guilabel:`(global filter)`
+.. |drag-handle| replace:: :raw-html:`<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 4 16" fill="currentColor"><circle cx="2" cy="3.5" r="1"></circle><circle cx="2" cy="6.5" r="1"></circle><circle cx="2" cy="9.5" r="1"></circle><circle cx="2" cy="12.5" r="1"></circle></svg>` :guilabel:`(drag handle)`
+
+Use *global filters* in Odoo Spreadsheet to create dynamic views of :doc:`inserted data <insert>` by
+mapping data source fields to global filters.
```

Comment:
As it's written the meaning comes across in a circular way, I'd try to be more concise and linear, if possible.
```suggestion
Create dynamic views of :doc:`inserted data <insert>`, by mapping data source fields to *global
filters* in the Odoo Spreadsheets app.
```